src/azul/service/avro_pfb.py

```diff
@@ -192,7 +192,10 @@ def _replace_null_with_empty_string(cls, object_json: AnyJSON) -> AnyMutableJSON
 
     def to_json(self, relations: Iterable['PFBRelation']):
         return {
-            **attr.asdict(self),
+            'id': self.id,
+            'name': self.name,
+            # https://fastavro.readthedocs.io/en/latest/writer.html#using-the-tuple-notation-to-specify-which-branch-of-a-union-to-take
```

Comment:
Shame this heading is so long, should the version stamp in the URL be pinned from `latest` to `1.4.4` so if the generated doc heading changes we can still find the section we need?

Reply:
That would be ideal, but unfortunately, they don't publish pinned versions of their documentation.